Makefile

```diff
@@ -140,4 +143,5 @@ ifndef ES_PASSWD
 	$(error ES_PASSWD is not set)
 endif
 	docker run -e ES_PASSWD="$(ES_PASSWD)" -e GOOGLE_APPLICATION_CREDENTIALS=./spacemesh.json -it go-spacemesh-python pytest -s test_bs.py --tc-file=config.yaml --tc-format=yaml
+	docker run -e ES_PASSWD="$(ES_PASSWD)" -e GOOGLE_APPLICATION_CREDENTIALS=./spacemesh.json -it go-spacemesh-python pytest -s hare/test_hare.py --tc-file=hare/config.yaml --tc-format=yaml
```

Comment:
You should probably make a script to run this for all tests. All tests should have the same structure.